web/client/simple.json

```diff
@@ -90,12 +90,19 @@
 		}, {
 			"name": "Help"
 		}, {
+			"name" : "Share",
+			"cfg": {
+				"closeGlyph" : "1-close"
+		}
```

Comment:
Credo non serva il file simple.json
Il 05/lug/2016 18:26, "Lorenzo Natali" <notifications@github.com> ha
scritto:

> In web/client/simple.json
> <https://github.com/geosolutions-it/MapStore2/pull/808#discussion_r69594312>
> :
>
> > @@ -90,12 +90,19 @@
> >  		}, {
> >  			"name": "Help"
> >  		}, {
> > +			"name" : "Share",
> > +			"cfg": {
> > +				"closeGlyph" : "1-close"
> > +		}
>
> indentation
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/geosolutions-it/MapStore2/pull/808/files/261b03642855cc4743c487f126be075a18dee52c#r69594312>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ALb5hNs3B-EeK5HmCh6Duq80Tcb1kIdqks5qSoW0gaJpZM4JFM1j>
> .
>